packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -239,14 +239,14 @@ describe('common/sqlite', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n'))
       .toMatchInlineSnapshot(`
 
-                                                                          // *** WARNING ***
-                                                                          // 
-                                                                          // These models were enriched with \`@@map\` information taken from the previous Prisma schema.
-                                                                          // - Model "AwesomeNewPost"
-                                                                          // - Model "AwesomeProfile"
-                                                                          // - Model "AwesomeUser"
-                                                                          // 
-                                                `)
+                                                                                                        // *** WARNING ***
+                                                                                                        // 
+                                                                                                        // These models were enriched with \`@@map\` information taken from the previous Prisma schema.
+                                                                                                        // - Model "AwesomeNewPost"
+                                                                                                        // - Model "AwesomeProfile"
+                                                                                                        // - Model "AwesomeUser"
+                                                                                                        // 
+                                                                    `)
```

Comment:
Is the whitespace diff significant?

Reply:
Not at all, it's annoying though that it changes